environment/lib/config.js

```diff
@@ -64,8 +64,12 @@ class RainbowConfig {
     static getParam (name) {
         const res = this.maybeGetParam(name);
         if (res === null) {
-            console.error(`Parameter ${name} must be specified.`);
-            process.exit(1);
+            if (!this.optionalValues.includes(name)) {
```

Comment:
This seems unnecessary. The code here already specifies that these arguments have default value as an empty string: https://github.com/near/rainbow-bridge/blob/b9e2f475fcd2e1250f3b36020511bdd531369385/environment/index.js#L91

Reply:
Without revert this change prepare gives error `Parameter bridge-src must be specified`, will check maybeGetParam

Reply:
I see the problem, `''` is treat as false in javascript, and so maybeGetParam returns false, will change maybeGetParam to check `undefined` to tell "parameter given (can be empty)"

Reply:
Thanks